src/libsystemd/sd-journal/journal-verify.c

```diff
@@ -882,6 +878,28 @@ int journal_file_verify(
                 goto fail;
         }
 
+        data_fp = fdopen(data_fd, "w+");
+        if (!data_fp) {
+                r = log_error_errno(-errno, "Failed to open data file stream: %m");
```

Comment:
no need to negate `errno`,  the log_error_errno() interalls does -abs() of the passed value anyway, i.e. is happy both with positive and negative errors, and considers them equivalent on input